### ISSUE TITLE
Errors when using ESM build of overlayscrollbars-vue

### ISSUE THREAD

**Issue by Magiczne** [KingSora/OverlayScrollbars#482]
**Describe the bug**
When building application using webpack following error shows in build window:
```
WARNING in ./node_modules/overlayscrollbars-vue/dist/overlayscrollbars-vue.esm.js 44:10-33
export 'default' (imported as 'OverlayScrollbars') was not found in 'overlayscrollbars' (possible exports: ClickScrollPlugin, OverlayScrollbars, ScrollbarsHidingPlugin, SizeObserverPlugin)
```

**Expected behavior**
Working ESM build of `overlascrollbars-vue`

**Comment by KingSora** [comment]
@Magiczne can you tell me the used version of all `overlayscrollbars` packages you use?

**Comment by Magiczne** [comment]
@KingSora sorry, sure

overlayscrollbars: 2.0.1
overlayscrollbars-vue: 0.5.0
webpack: 5.75.0
webpack-cli: 5.0.0

**Comment by KingSora** [comment]
@Magiczne Thats interesting.. can you please wipe your `node_modules` folder and install everything again? Please also wipe any cache which webpack could use to mix up package versions.

To me it looks like you have an old version of `overlayscrollbars-vue`. Since `v0.4.0` the package has no `dist` folder. [This is how the package structure looks like since `v0.4.0`](https://www.jsdelivr.com/package/npm/overlayscrollbars-vue?version=0.5.0).

**Comment by Magiczne** [comment]
@KingSora At time I wrote this post I've already tried wiping node modules and so on, but with no results, including clearing Yarn cache, which caches packages locally.

For some reason the version in lock was 0.5.0, and the version installed localle was 0.4.0 (wth?).

I've managed to fix this by downgrading `overlayscrollbars-vue` to `0.4.0` without the `^` in the package.json to force this version and then manually bumping and adding caret sign again.

No slightest idea what could've been the cause of this, as I cannot reproduce it again in any circumstance, but maybe if someone encounters the same kind of problem this "solution" will help them.

Thanks for your time
